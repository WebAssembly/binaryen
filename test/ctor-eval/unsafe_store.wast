(module
  (memory 256 256)
  (data (i32.const 10) "waka waka waka waka waka")
  (export "test1" $test1)
  (export "test2" $test2)
  (export "test3" $test3)
  (func $test1
    (i32.store8 (i32.const 9) (i32.const 109)) ;; before first segment
  )
)
