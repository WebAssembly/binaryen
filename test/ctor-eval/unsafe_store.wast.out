(module
 (type $0 (func))
 (memory $0 256 256)
 (data (i32.const 0) "\00\00\00\00\00\00\00\00\00mwaka waka waka waka waka")
 (export "test1" (func $test1))
 (export "test2" (func $test2))
 (export "test3" (func $test3))
 (func $test1 (type $0)
  (nop)
 )
)
